src/cleanup_rules/swift/scope_config.toml

```diff
@@ -25,21 +25,52 @@ generator = """(
 name = "Function"
 [[scopes.rules]]
 matcher = """(
-    (function_declaration
-        name: (simple_identifier)? @name
-        (parameter)? @params
-        body: (function_body)
-    ) @function
+    [
+        (function_declaration
+            name: (simple_identifier) @m_function_with_param_name
+            (parameter) @m_function_with_param_params
+            body: (function_body)
+        )
+        (function_declaration
+            name: (simple_identifier) @m_function_without_param_name
+            .
+            body: (function_body)
```

Comment:
why are we matching the body to create the scope?
The body might change due to some `PARENT` edits.